server/math_server/routes/users.js

```diff
@@ -176,16 +176,70 @@ exports.list = function(req, res){
 }
 
 //Get the HighScores
+/*
+{
+        "_id": "54d1d9a823883d0300e6456a",
+        "fbID": "987654321",
+        "firstName": "test1",
+        "lastName": "test1pw",
+        "__v": 1,
+        "score": [1, 2, 3, 4],
+        "friends": [
+            "54d3bb8f643155030037eb75"
+        ],
+        "createdAt": "2015-02-04T08:34:48.669Z"
+    }
+
+
+currently - [1, 2, 3, 4]
+obj = { 
+   "user": fbid,
+   "highscore": .... 
+}
+
+res.end(JSON.stringify(obj))
+*/
+
+
+/*Array.max = function(array){
+  return Math.max.apply(Math, array);
+
+};
+*/
+
+/*function highestScore(array)
+{
+    var m = -Infinity, i = 0, n = a.length;
+
+    for (; i != n; ++i) {
+        if (a[i] > m) {
+            m = a[i];
+        }
+    }
+    return m;
+}
+*/
+
+
+
 exports.getHighScores = function(req,res){
```

Comment:
Hi Kevin! Get requests shouldn't modify/update anything.  you should only be doing a findOne, which returns a user object.  When you get this user object, you're given an array of scores.  You need to be able to return the highest score in this array of scores